lua/lean/lean3.lua

```diff
@@ -31,7 +31,7 @@ local _GOAL_MARKER = vim.regex('⊢ .\\{-}\n\\(\\s\\+.\\{-}\\(\n\\|$\\)\\)*\\zs'
 
 --- Detect whether the current buffer is a Lean 3 file.
 function lean3.__detect()
-  local path = vim.api.nvim_buf_get_name(0)
+  local path = vim.fn.bufname("%")
```

Comment:
Figured we should play it safe since I don't think we're `schedule_wrap`'ed here.